dipy/io/vtk.py

```diff
@@ -146,15 +146,19 @@ def save_vtk_streamlines(streamlines, filename,
     polydata.SetPoints(vtk_points)
     polydata.SetLines(vtk_lines)
 
-    writer = vtk.vtkPolyDataWriter()
-    writer.SetFileName(filename)
-    writer = utils.set_input(writer, polydata)
+    file_extension = filename.split(".")[-1].lower()
```

Comment:
I think, it would be better to use `os.path.splitext()`. it is more explicit

Reply:
i have made the corresponding change